packages/new-polymath-ui/src/components/FormItem/Input/index.tsx

```diff
@@ -8,6 +8,8 @@ export interface Props
   FormikComponent: React.ComponentType<any>;
   placeholder?: string;
   component: React.ComponentType<FormikExternalProps>;
+  unit?: string;
+  label?: string;
```

Comment:
@RafaelVidaurre here we face an issue with the typings: as this component takes a `component` prop that can be any possible component, and forward the props, it means TS complains if you apply any prop that is not in the interface of `FormItem.Input` itself. ex: `<FormItem.Input component={Select} someObscureSelectProp="fu" />` it will say that `someObscureSelectProp` is missing from `FormItem.Input` interface.
Here Bilal added the props to the interface but that's not scalable...Do you know a solution for this?

Reply:
I'll have a look, though I have my suspicions as to why this could be. Will get back to you in a bit